Gemfile.lock

```diff
@@ -1,165 +1,166 @@
 GEM
```

Comment:
This file can probs be .gitignored

Reply:
`Gemfile.lock` is intended to be committed to git; it provides pinning of specific known-working versions of gems within the range allowed by the gemfile.